datumaro/plugins/cityscapes_format.py

```diff
@@ -110,19 +117,54 @@ class CityscapesPath:
     LABELMAP_FILE = "label_colors.txt"
 
 
+DEFAULT_BACKGROUND_LABEL = "background"
+DEFAULT_BACKGROUND_COLOR = (0, 0, 0)
+
+
+def has_colors(self) -> bool:
```

Comment:
Probably shouldn't use `self` as a name outside of a class.

Reply:
Fixed